app/controllers/comments_controller.rb

```diff
@@ -19,4 +27,12 @@ def create
   def comment_params
     params.require(:comment).permit(:text)
   end
+
+  def destroy
+    @comment = Comment.find(params[:id])
+    @post = @comment.post
+    @post.decrement!(:comments_counter)
+    @comment.destroy!
+    redirect_to user_post_path(id: @post.id), notice: 'The comment has been successfully removed!'
+  end
```

Comment:
- [ ] Also when I try to delete a comment I get an error, kindly fix that 🙃 🙃 

<img width="759" alt="Screenshot 2023-11-10 at 14 10 18" src="https://github.com/Marcraphael12/rails-blog-app/assets/74501523/daa71a2f-3e70-46f9-a9cd-77ed02a84033">